src/presentation-markup.html

```diff
@@ -6135,13 +6126,15 @@ <h5>Attributes</h5>
     of <code class="attribute">accent</code> for <code class="element">mover</code>.  If
     <em>overscript</em> is an embellished operator, the <code class="attribute">accent</code> attribute of the <code class="element">mo</code>
     element at its core is used as the default value.</p>
+    <p>[[Mathml-Core]] does not support the <code class="attribute">accent</code> attribute on <a href="#presm_mo"></a>.
+      For compatibility with MathML-core, the <code class="attribute">accentunder</code> should be set on <code class="element">munder</code>.</p>
  
    </section>
  
    <section class="fold">
-    <h5>Examples</h5>
+    <h5 id="id="presm_mover-ex"">Examples</h5>
  
-    <p>The MathML representation for the examples shown above is:</p>
+    <p>Two examples demonstrating how <code class="attribute">accentunder</code> affects rendering:</p>
```

Comment:
they don't use `accentunder`, did you mean `accent` here ?

Reply:
Fixed